dnf-plugins/py3query.py

```diff
@@ -318,9 +318,20 @@ def run(self):
                             pkg.suggests + pkg.supplements + pkg.enhances):
                 require = str(require).split()[0]
                 requirement = all_provides.get(require)
-                if (is_unversioned(require) and requirement and not
-                        (require.endswith('-doc') or python_versions.get(requirement) == {3})
-                        and require != 'python-unversioned-command'):
+                if (
+                    is_unversioned(require)
+                    and requirement
+                    and not (
+                        require.endswith('-doc')
+                        or python_versions.get(requirement) == {3}
+                    )
+                    and not require.endswith('-wheel')
+                    and require not in (
+                        'python-rpm-macros',
+                        'python-srpm-macros',
+                    )
+                    and require != 'python-unversioned-command'
```

Comment:
can wee merge this with above `not in`, or am i missing something?

Reply:
Also, maybe we can use: https://github.com/fedora-python/taskotron-python-versions/blob/24a3a410d714814d94b769a204a42487eadd23dc/taskotron_python_versions/two_three.py#L35

Reply:
Ah! That's what I was looking for.
It doesn't include `python-unversioned-command` or `-doc`, so keeping those here.